ia/src/protocole.py

```
@@ -107,8 +131,15 @@
         "mape_size": [0, 0],
         "commandes": [],
         "inventory": {},
+        "last_look": [],
         "level": 1,
         "is_alive": True,
+        "move": {
+            "consecutive_turns": 0,
+            "forward": False,
+            "distance_to_food": -1,
+            "last_food": None,
+        }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)